models/spleen_ct_segmentation/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.5.2",
+    "version": "0.5.3",
     "changelog": {
+        "0.5.3": "test new ci",
```

Comment:
I think this code is temporary test, please remember to remove it before merging this PR.